sites/NOAA_GFDL/mdtf_gfdl.csh

```diff
@@ -134,53 +140,58 @@ end
 ## clean up tmpdir
 wipetmp
 
-## run the command (unbuffered output)
-echo 'script start'
-/usr/bin/env python -u "${REPO_DIR}/src/mdtf_gfdl.py" \
---frepp \
+## run the command
+echo "mdtf_gfdl.csh: conda activate"
+source "${REPO_DIR}/src/conda/conda_init.sh" -q "/home/mdteam/anaconda"
```

Comment:
/home/mdteam/anaconda should be changed to the newer miniconda3 package installed in /home/oar.gfdl.mdtf: `/home/oar.gfdl.mdtf/miniconda3`

Reply:
Great! Thanks for setting conda up in the new role account!

Reply:
Fixed in 0bcae3c94530d13df7ff41972b38f26937af9242